setup/common/003_solaris_cert_fix.rb

```diff
@@ -0,0 +1,54 @@
+test_name 'Add digicert to solaris keystore'
+
+# Only need to run this on soliars 11, 11.2, and sparc
+skip_test 'No solaris 11 version that needed keystore updating' if ! hosts.any? { |host| host.platform=~ /solaris-11(\.2)?-(i386|sparc)/}
+
+solaris_hosts = Array.new
+
+hosts.each do |host|
+  solaris_hosts.push(host) if host.platform=~ /solaris-11(\.2)?-(i386|sparc)/
+end
+
+step 'Download the DigiCert Trusted Root G4' do
+  solaris_hosts.each do |host|
+    on(host, 'curl --remote-name https://cacerts.digicert.com/DigiCertTrustedRootG4.crt.pem')
```

Comment:
Since the root cert isn't going to change over time, I'd just hard code this similar to how handle windows in https://github.com/voxpupuli/beaker-puppet/blob/8614a6337014438ebe91f4eacfabc7098a8e261d/setup/git/070_InstallCACerts.rb#L82-L84